src/components/Congress/index.js

```diff
@@ -11,8 +12,26 @@ import Wallet from './Wallet';
 import { CongressAddLegislation } from '../Voting/Referendum/ProposalForms/CongressAddLegislation/CongressAddLegislation';
 // eslint-disable-next-line max-len
 import { CongressAddLegislationViaReferendum } from '../Voting/Referendum/ProposalForms/CongressAddLegislationViaReferendum/CongressAddLegislationViaReferendum';
+import { congressActions, officesActions } from '../../redux/actions';
+import { congressSelectors, officesSelectors } from '../../redux/selectors';
 
 function Congress() {
+  const dispatch = useDispatch();
+  const pallets = useSelector(officesSelectors.selectorPallets);
+  const congresAccountAddress = useSelector(congressSelectors.walletAddress);
+
+  useEffect(() => {
+    dispatch(officesActions.getPalletIds.call());
+  }, [dispatch]);
+
+  useEffect(() => {
+    dispatch(congressActions.congressGetWallet.call());
+  }, [dispatch, pallets]);
+
+  useEffect(() => {
+    dispatch(congressActions.congressGetAdditionalAssets.call());
```

Comment:
i really dont like this pattern of using useEffects to make a chained of calls one after the other.

Is this really an accepted pattern @kacperzuk-neti @SzymonZiebaNeti ?

Reply:
I'm not a Redux Sagas expert, but I don't like it either. it should be combined in a single action (which, if needed, can dispatch actions it depends on its of in the saga if needed).